doc/nrf/device_guides/working_with_nrf/nrf70/developing/fw_patches_ext_flash.rst

```diff
@@ -0,0 +1,133 @@
+.. _ug_nrf70_developing_fw_patch_ext_flash:
+
+Firmware patches in the external flash
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Overview
+********
+
+The nRF70 Series firmware patch is memory intensive and is by default part of the internal flash.
+In order to save flash memory for resource constrained hosts, the nRF70 Series offers a feature to move firmware patches to the external flash, if the external flash is available.
+
+Pre-requisites
+--------------
+
+The following conditions must be met to use this feature:
+
+- The external flash must be available and configured in the device tree.
+- The external flash must be large enough to store the firmware patches, maximum size is 128 KB.
+
+Constraints
+-----------
+
+The following constraints apply to this feature:
```

Comment:
I think the Constraints section could be better named "Limitations". Also, better if hey could be converted to notes.

Reply:
Wouldn't that be too many notes?

Reply:
Yes, it would be better if it is converted into a note to prevent the redundancy.
 
Perhaps?

"Currently, in Nordic Semiconductor SoCs, access to external memory and the Execute in Place (XIP) feature are available through the QSPI interface only."